backend/app.js

```diff
@@ -171,16 +171,16 @@ socketIO.on('connection', (socket) => {
               OpenGame.find({ isOpen: true }, (err, allOpenGames) => {
                 socketIO.sockets.emit('newGameInfo', allOpenGames);
               });
+            }).then(() => {
+              OpenGame.find({ _id: player2info.gameId }, (err, game) => {
+                socket.join(player2info.gameId);
+                console.log(game);
```

Comment:
Yes, looks good, but console,log is not needed anymore

Reply:
I agree, remove it.

Reply:
sorry)